### PR TITLE
[MOBILE-946] prep flutter plugin for initial publish

### PR DESCRIPTION
**Pull request by crow**
### What do these changes do?
Updates the flutter plugin to prepare for publish.

### Why are these changes necessary?
To push the plugin to pub.dev.

### How did you verify these changes?
Ran publish dry run via
`flutter pub pub publish --dry-run`

#### Verification Screenshots:
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,26 @@
...
+2. Install your flutter package dependencies by running the following in the command line at your project's root directory:
 
-This plugin is not yet published, so you will have to use the current master repo as a [dependency in your pubspec.yaml file](https://flutter.io/platform-plugins/).
 ```
-  airship:
-    git: git://github.com/urbanairship/airship-flutter.git
+$ flutter pub get
+```
+
+3. Import airship into your project:
```
Comment (BrianBatchelder) [inline README.md]: Is there a particular source file this needs to go in? Maybe it's obvious to Flutter developers, or maybe more detail would be helpful.

Comment (crow) [reply README.md]: no, this is the only one, there is also an example below with this re-iterated

File: README.md [hunk 2]
```diff
@@ -1,29 +1,26 @@
...
+1. Add the airship dependency to your package's pubspec.yaml file:
 
-Feature Status:
-- [x] Tags
-- [x] Notifications
-- [x] Deep links
-- [x] Inbox
-- [x] Named User
-- [x] Push Events
-- [ ] Custom events
```
Comment (marc-scig) [inline README.md]: are these non-checked items finished? if not do we have tickets for them?

Comment (crow) [reply README.md]: They are finished

**Pull request merged**